### PR TITLE
Fix get api v1 playlists closes#5

### PR DESCRIPTION
**Pull request by andrewetobin**
__x_ Wrote Tests
__x_ Implemented
__x_ Reviewed
# Neccesary checkmarks:
- [x] All Tests are Passing

- [x] The code will run locally

## Type of change
- [x] New feature
- [] Bug Fix

# Implements/Fixes:
fixes the functionality for api/v1/playlists. renders correct response.
closes #

# Check the correct boxes

- [x] This broke nothing
- [] This broke some stuff
- [] This broke everything

# Testing Changes
- [] No Tests have been changed
- [x] Some Tests have been changed
- [] All of the Tests have been changed(Please describe what in the world happened)

# Checklist:

- [] My code has no unused/commented out code
- [x] I have reviewed my code
- [] I have commented my code, particularly in hard-to-understand areas
- [x] I have fully tested my code

# Please Include a link to a gif of your feelings about this branch
Alt Text

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -91,15 +91,30 @@ app.patch('/api/v1/songs/:id', (request, response) => {
...
-    .then((playlists) => {
-      response.status(200).json(playlists);
-    })
-    .catch((error) => {
-      response.status(500).json({ error });
-    });
+  let playlists = []
+  let songs = []
+  database('playlists').select(['playlists.id', 'playlists.playlist_name'])
+  .then((a) => {
```
Comment (KathleenYruegas) [inline server.js]: what does 'a' represent?

File: server.js [hunk 2]
```diff
@@ -91,15 +91,29 @@ app.patch('/api/v1/songs/:id', (request, response) => {
...
+  database('playlists').select(['playlists.id', 'playlists.playlist_name'])
+  .then((allPlaylists) => {
+    playlists = allPlaylists
+  })
+  database("songs")
+  .select(['songs.id', 'name', 'artist_name', 'genre', 'song_rating', 'playlist_songs.playlist_id'])
+  .join("playlist_songs", 'songs.id', 'playlist_songs.song_id')
+  .then((allSongs) => {
+    songs = allSongs;
+    for(let playlist of playlists) {
```
Comment (KathleenYruegas) [inline server.js]: @ameseee Does knex not have something to allow me to access the associated songs with a playlist?  Or we need to do this manually?  Please review, thanks!

**Comment by KathleenYruegas** [comment]
@ameseee Tagging you for review!  I left another comment above as well!  So does Knex not give us the relationships as easily as activerecord in rails does?

**Comment by ameseee** [comment]
Hi @KathleenYruegas - you are correct, knex doesn't give you nearly as much as activerecord, so we're stuck doing a lot more manually.

**Pull request merged**